api/teams_test.go

```diff
@@ -636,6 +733,17 @@ var _ = Describe("Teams API", func() {
 							Expect(teamServerDB.CreateTeamCallCount()).To(Equal(1))
 						})
 					})
+
+					Context("when passed Generic OAuth credentials", func() {
+						BeforeEach(func() {
+							team.GenericOAuth = &genericOAuth
+						})
+
+						It("updates the Generic OAuth auth for that team", func() {
+							Expect(response.StatusCode).To(Equal(http.StatusCreated))
+							Expect(teamServerDB.CreateTeamCallCount()).To(Equal(1))
```

Comment:
Hmm. Is this actually covering the right thing? It doesn't make any assertions about the auth bit. AFAICT that method (`CreateTeam`) is called regardless; `UpdateGenericOAuth` is what to look for. Looks like the other tests might be similarly broken.